src/tyr/route_serializer_osrm.cc

```diff
@@ -1563,7 +1508,7 @@ json::ArrayPtr serialize_legs(const google::protobuf::RepeatedPtrField<valhalla:
     // Get a summary based on longest maneuvers.
     double duration = leg->summary().time();
     double distance = units_to_meters(leg->summary().length(), !imperial);
-    output_leg->emplace("summary", summarize_leg(leg));
+    output_leg->emplace("summary", leg_summaries[leg_index]);
```

Comment:
`output_leg` is an `unordered_map` under the hood so it's better to move arguments

Reply:
I'm not sure what you're suggesting.  As you say output_leg is an unordered_map - and the emplace_back() method will use the move operator if possible.  What is being passed to it is a std::string, which has a move operator.

Within CLion I did a "go to definition" on that line of code and sure enough, it takes me to:
`pair<iterator, bool> emplace(_Args&&... __args)`

I believe that means the move operator will be employed.

Reply:
yeah, exactly. I believe so as well

Reply:
thanks